src/components/Slider/BootstrapSlider.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Slider from 'bootstrap-slider';
+
+class BootstrapSlider extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this._slider = null;
+  }
+
+  componentDidMount() {
+    const that = this;
+
+    this._slider = new Slider(this.sliderDiv, {
+      ...this.props
+    });
+    this._slider.on('slide', value => that._onSlide(value));
```

Comment:
@sharvit can you remind me why we chose to use `bindMethods` syntax for binding class functions? It has been a long time since that was discussed...

to @dmiller9911 's point, I believe many communities have moved to the `classMethod = () => {}` syntax...